packages/website/src/components/Schedule/ScheduleContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+
+import { Center } from '@coderscamp/ui/components/Center';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+interface ScheduleContainerProps {
+  title: string;
+  children: ReactElement;
+}
+
+export const ScheduleContainer = ({ title, children }: ScheduleContainerProps) => {
+  return (
+    <Center flexDirection="column" mx="auto" w="100%" mt="120px" mb="80px">
```

Comment:
🤔 Total margin-top and margin-bottom of this section should be 80px each. You can remove `mt` and `mb` here and replace them with `my="80px"`. Additionaly you will need to change `m="64px 0"` below to `mb="64px"`